torch/csrc/jit/codegen/cuda/scheduler/registry.cpp

```diff
@@ -895,6 +895,13 @@ class PersistentKernelScheduler : public SchedulerEntry {
   }
 
   static bool canScheduleCompileTime(Fusion* fusion) {
+    auto welford_ops = findReductionOps<WelfordOp>(fusion);
```

Comment:
Since we check if any schedulers can run the kernel without segmentation, I don't see any path for this to get translated from welford, because only the segmenter will attempt to translate welford.